CONFIGURATION.md

```diff
@@ -0,0 +1,128 @@
+# Configuring NTPD-rs
+
+NTPD-rs is an implementation of NTP completely written in Rust, with a focus on exposing a minimal attack surface. The project is currently in an early stage, but if you want to try out NTPD-rs on a non-critical system, this guide provides the basic information needed to properly build and configure NTPD-rs.
+
+## Limitations
+
+The current implementation has several important limitations which it is important to be aware of. First of all, the current implementation is an NTP client only, and does not support acting as an NTP server.
+
+Second, there are currently a few limitations around configuring connections to other NTP nodes:
+ - Only Client-Server connections are supported
+ - DNS lookup is currently only done at startup. Changes in the IP address 
+ - There is no support for NTP pools yet. Multiple 
+ - Changes in network interfaces are not picked up dynamically and will require a reboot of the daemon.
+
+Finally, please be aware that this is very early stage software, and is not yet ready for production software. This software SHOULD NOT be used on any system where you cannot handling experiencing issues, either reliability or security related.
+
+## Building
+
+Currently, NTPD-rs only support linux based operating systems. Our current testing only targets linux kernels after version 5.0.0, older kernels may work but are not guaranteed.
```

Comment:
"support" -> "supports", and I'd capitalize Linux.

Is there some particular reason for taking 5.0.0 as the minimum bound here? In particular, my server is running 4.19, and so does Debian 10 (Buster), which AFAICT is still in fairly widespread usage.

Reply:
I think it would be valuable to spend ~30 minutes on research to see what limitations there could be in supporting older kernels and writing up the result of that, as in "Based on our understanding of API support, we think this is supported on kernels after 3.15, but we have only tested on 5.x and later."